src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLStoptimeImpl.java

```diff
@@ -13,7 +13,7 @@
 
   @Override
   public DataFetcher<Object> stop() {
-    return environment -> getRoutingService(environment).getStopForId(getSource(environment).getStopId());
+    return environment -> getSource(environment).getStop();
```

Comment:
Presumably this returned the wrong thing before this change, isn't it?

Reply:
It worked correctly, but it is unnecessary to return the id and fetch the same object again from the index